openquakeplatform_taxtweb/bin/check_taxonomy_links.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+ddown_items="$(grep push $(find . -name '*.js') | grep dataGemHelp | sed "s/.*gem_taxonomy_base + '//g;s/'.*//g" | sort | uniq)"
+
+# 'direction-x', 'direction-y'
+# from views.py:  sub1help = [taxonomy_base + 'direction-x', taxonomy_base + 'direction-y']
```

Comment:
For me this comments are useless

Reply:
This is a reference where direction x and y are defined, not commented code